microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -28,52 +28,32 @@ In IE mode, the rendering process is based on Internet Explorer 11.  The Microso
 
 IE mode is currently available on Windows 10 Version 1903 \(May 2019 Update\), but is coming soon to all supported Windows platforms.  
 
-## Launching the DevTools on a tab in IE mode  
+## Open the DevTools on a tab in IE mode  
 
-If you are trying to view the document mode of a web site in IE mode, choose the badge in the address bar.  
+To view the document mode of a web site in IE mode, select the badge in the address bar.  
 
 :::image type="complex" source="../media/ie-mode-badge-doc-mode.msft.png" alt-text="View document mode using IE mode badge" lightbox="../media/ie-mode-badge-doc-mode.msft.png":::
    View document mode using IE mode badge  
 :::image-end:::  
 
-If a tab is using IE mode, the DevTools do not work and the following conditions occur.
-
-*   If you select `F12` or select `Ctrl`+`Shift`+`I`, a blank instance of the Microsoft Edge \(Chromium\) DevTools is launched displays the following message.  
-    
-    ```text
-    Developer Tools are not available in Internet Explorer mode.  To debug the page, open it in Internet Explorer 11.
-    ```  
+If a tab uses IE mode, a number of DevTools, such as **Network** and **Performance**, do not work because the rendering engine switches from Chromium to Internet Explorer 11. You'll also notice the following conditions.
     
-*   If you open a contextual menu \(right-click\) and choose **View Source**, Notepad is launched.  
-*   If you open a contextual menu \(right-click\), the **Inspect Element** is not visible.  
-
-The reason that a number of the tools within the DevTools \(like the **Network** and **Performance** tools\) do not work is the rendering engine switches from Chromium to Internet Explorer 11 in IE mode.  To provide feedback, navigate to [Getting in touch with the Microsoft Edge DevTools team](#getting-in-touch-with-the-microsoft-edge-devtools-team).  
+*   Opening the contextual menu \(right-click\) and selecting **View Source** launches Notepad.  
+*   **Inspect Element** is not visible on the contextual menu \(right-click\).  
+*   Selecting `F12` or `Ctrl`+`Shift`+`I` opens a blank instance of the Microsoft Edge \(Chromium\) DevTools and displays the following message: **Developer Tools are not available in Internet Explorer mode.  To debug the page, open it in Internet Explorer 11.**  
 
 :::image type="complex" source="../media/ie-mode-devtools.msft.png" alt-text="DevTools launched in IE mode" lightbox="../media/ie-mode-devtools.msft.png":::
```

Comment:
@Reezaali - Finding: this image was updated Aug 26, 2021.
https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/devtools-guide-chromium/media/ie-mode-devtools.msft.png